mvcc/backend/backend.go

```diff
@@ -179,15 +179,17 @@ func (b *backend) Size() int64 {
 
 func (b *backend) run() {
 	defer close(b.donec)
-
+	t := time.NewTimer(b.batchInterval)
+	defer t.Stop()
 	for {
 		select {
-		case <-time.After(b.batchInterval):
+		case <-t.C:
 		case <-b.stopc:
 			b.batchTx.CommitAndStop()
 			return
 		}
 		b.batchTx.Commit()
+		t.Reset(b.batchInterval)
```

Comment:
Thanks! But isn't it other way round? If `.Reset()` returns `false`, it means that timer has expired and channel is full (with one value), otherwise (if timer is still active), it means that channel has not yet been written to?